components/schedule-editor/README.md

```diff
@@ -26,22 +26,22 @@ Nylas Schedule Editor is currently in active development. Want to contribute? [F
 
 ## Documentation
 
-- [Nylas Scheduler reference](https://developer.nylas.com/docs/user-experience/components/scheduler-editor-component/)
+- [Nylas Booking reference](https://developer.nylas.com/docs/user-experience/components/scheduler-editor-component/)
```

Comment:
```suggestion
- [Nylas Booking reference](https://developer.nylas.com/docs/user-experience/components/schedule-editor-component/)
```

Reply:
This is a change we should make but not until docs are updated -- right now, committing this change would result in a 404

Reply:
The docs don't have those links yet any how. Safe to update.